packages/nsq/nsq.0.1.1/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/nsq-ocaml"
+bug-reports: "https://github.com/ryanslade/nsq-ocaml/issues"
+dev-repo: "https://github.com/ryanslade/nsq-ocaml.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build}
+  "containers"
+  "lwt"
+  "ocplib-endian"
+  "integers"
+]
+available: [ocaml-version >= "4.03.0"]
```

Comment:
Alas, failed on OCaml 4.06.0 

```
#=== ERROR while installing nsq.0.1.1 =========================================#
# opam-version         1.2.2 (aa258ecc06d3aea5a67f442a4ffd23f2a457180b)
# os                   linux
# command              jbuilder build -p nsq -j 2
# path                 /home/opam/.opam/4.06.0/build/nsq.0.1.1
# compiler             4.06.0
# exit-code            1
# env-file             /home/opam/.opam/4.06.0/build/nsq.0.1.1/nsq-18-e93b6f.env
# stdout-file          /home/opam/.opam/4.06.0/build/nsq.0.1.1/nsq-18-e93b6f.out
# stderr-file          /home/opam/.opam/4.06.0/build/nsq.0.1.1/nsq-18-e93b6f.err
### stderr ###
# [...]
#     ocamlopt src/nsq.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06.0/bin/ocamlopt.opt -w -40 -g -I /home/opam/.opam/4.06.0/lib/bytes -I /home/opam/.opam/4.06.0/lib/containers -I /home/opam/.opam/4.06.0/lib/integers -I /home/opam/.opam/4.06.0/lib/lwt -I /home/opam/.opam/4.06.0/lib/ocaml -I /home/opam/.opam/4.06.0/lib/ocplib-endian -I /home/opam/.opam/4.06.0/lib/result -no-alias-deps -I src -o src/nsq.cmx -c -impl src/nsq.ml)
# File "src/nsq.ml", line 230, characters 41-49:
# Error: This expression has type bytes but an expression was expected of type
#          string
#       ocamlc src/nsq.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06.0/bin/ocamlc.opt -w -40 -g -bin-annot -I /home/opam/.opam/4.06.0/lib/bytes -I /home/opam/.opam/4.06.0/lib/containers -I /home/opam/.opam/4.06.0/lib/integers -I /home/opam/.opam/4.06.0/lib/lwt -I /home/opam/.opam/4.06.0/lib/ocaml -I /home/opam/.opam/4.06.0/lib/ocplib-endian -I /home/opam/.opam/4.06.0/lib/result -no-alias-deps -I src -o src/nsq.cmo -c -impl src/nsq.ml)
# File "src/nsq.ml", line 230, characters 41-49:
# Error: This expression has type bytes but an expression was expected of type
#          string
```

Needs a constraint on < 4.06.0 here as a result

Reply:
I've updated the PR and it now build if 4.06

4.02.3 fails though, but that should be ok because the constraint asks for at least 4.03